src/index.js

```diff
@@ -1,5 +1,6 @@
 import './style.css';
 import { Todolist } from '../modules/list.js';
+import { updateStatus, clearButton, checkboxListeners } from '../modules/status.js';
 
 const lists = new Todolist();
```

Comment:
@Alfredbis29 Kindly check again how you have set up the application, i am able to add the tasks by clicking the arrow icon.

Reply:
Hello @Koshao-sys sorry for delaying a bit , I hope you are doing well, I set up the app correctly I checked that more times but  was still not able to add tasks. All was set up correctly , if that works on your side now ,share a short video so that the next reviewer won't be confused 😐.

Reply:
Hello @Alfredbis29, I have recorded a video here: https://user-images.githubusercontent.com/92449654/220990837-6577c9e1-4e21-4db9-94b8-a4084b5cb7c6.mp4